tests/test_vtk.py

```diff
@@ -39,17 +40,45 @@ def test_validation_pluto(fn):
 @pytest.mark.parametrize(("fn", "geometry"), PLUTO_VTK_FILES)
 def test_parse_pluto_metadata(fn, geometry):
     ds = yt_idefix.load(fn)
-    assert ds._definitions_header == os.path.join(ds.fullpath, "definitions.h")
+    assert os.path.samefile(
+        ds._definitions_header, DATA_DIR.joinpath("pluto_sedov", "definitions.h")
+    )
     assert "unit_length" in ds.parameters
     assert "unit_density" in ds.parameters
     assert "unit_velocity" in ds.parameters
-    assert ds.current_time != -1
-    assert ds.geometry == "cartesian"
-    assert ds.length_unit / ds.velocity_unit == ds.time_unit
-    with pytest.raises(ValueError):
-        ds = yt_idefix.load(fn, definitions_header="definitions2.h")
-    with pytest.warns(UserWarning):
-        ds = yt_idefix.load(fn, definitions_header="definitions2.h", geometry=geometry)
+    assert ds.geometry == geometry
+    assert (
+        assert_allclose_units(ds.length_unit, unyt_quantity(3.08567758e21, "cm"))
+        is None
+    )
+    assert (
+        assert_allclose_units(ds.velocity_unit, unyt_quantity(100000.0, "cm/s")) is None
+    )
+    assert (
+        assert_allclose_units(ds.mass_unit, unyt_quantity(4.91416082e40, "g")) is None
+    )
+    assert (
+        assert_allclose_units(ds.time_unit, unyt_quantity(3.08567758e16, "s")) is None
+    )
+    assert (
+        assert_allclose_units(ds.magnetic_unit, unyt_quantity(4.58462477e-07, "G"))
+        is None
+    )
+    assert ds.current_time >= 0
+
+
+@pytest.mark.parametrize("fn", [_[0] for _ in PLUTO_VTK_FILES])
```

Comment:
No need to parametrize this test, I would rather hardcode the name of one dataset here.

Reply:
Okay. How about creating `PLUTO_SEDOV` at header for those tests where checking one dataset is enough, also in case future changes on `pluto_sedov`

Reply:
Sure, go ahead